test/integration/tags/raw_tag_test.rb

```diff
@@ -24,6 +24,7 @@ def test_open_tag_in_raw
   end
 
   def test_invalid_raw
-    assert_match_syntax_error /tag was never closed/, '{% raw } foo {% endraw %}'
```

Comment:
I would prefer to keep all three tests. It's not obvious to me that the new tests cover the exact same codepaths that the old test did.